org.opentosca.yamlconverter.yamlmodel/src/org/opentosca/yamlconverter/yamlmodel/yaml/element/Input.java

```diff
@@ -7,7 +7,7 @@
 public class Input extends YAMLElement {
 
 	private String type = "";
-	private String defaultValue = "%%USER_INPUT%%";
+	private String defaultValue = "%%USER_INPUT_REQUIRED%%";
```

Comment:
I understand, that would make it a little bit more complex to distinguish... I'll fix it and reset the value (I prefer empty string instead of null value)